src/corelibs/U2View/src/ov_msa/MaEditorSequenceArea.h

```diff
@@ -161,7 +161,8 @@ public slots:
 
 protected slots:
     void sl_changeColorScheme();
-    void sl_fillCurrentSelectionWithGaps();
+    void sl_insertGaps();
+    void sl_replaceWithGaps();
```

Comment:
nit: I would recommend to be more detailed in the method name: sl_replaceSelectionWithGaps. Same for the sl_insertGaps.

Reply:
OK